lib/Debugger.ProtocolHandler/DebuggerCallFrame.cpp

```diff
@@ -114,6 +114,7 @@ namespace JsDebug
             .setReturnValue(GetReturnValue())
             .setScopeChain(GetScopeChain())
             .setThis(GetThis())
+            .setUrl("")
```

Comment:
This is now required, but the data isn't readily available at this point in the code.